UniWatch/Views/Student/Enroll.cshtml

```diff
@@ -0,0 +1,38 @@
+﻿@model UniWatch.ViewModels.EnrollViewModel
```

Comment:
Will need search functionality working before this will be approved. Consult with Patrick